src/app/timeseries-menu.component.html

```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
       [showXAxisLabel]="showXAxisLabel"
       [showYAxisLabel]="showYAxisLabel"
       [xAxisLabel]="xAxisLabel"
-      [yAxisLabel]="yAxisLabel"
+      [yAxisLabel]="yValTitle()"
       [autoScale]="autoScale"
     >
     </ngx-charts-line-chart>
     <br />
     <button mat-raised-button (click)="createCsvFromTimeseriesData()">
       Download Data
     </button>
+    <br />
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
+    <p>Longitude: {{ this._model.settings.CurrGridBoxLon }}</p>
+    <br />
+    <br />
+    <button mat-raised-button (click)="closeTimeSeries()">
+      Close Time Series
```

Comment:
I don't like _Close Time Series_ button. I prefer how you put an X in the top right corner of the navbar to close that menu. Please do that here too, just add an X in the upper right corner of this window. I don't think we really need a close button, but that would look better.